### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by ozodsilver** [community/community#65456] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

элементы проходят над другими элементами когда количество столбцов меньше 12. 
![image](https://github.com/community/community/assets/71330578/2dc25eb3-5f53-4bde-9474-fcc3464f0431)
![image](https://github.com/community/community/assets/71330578/692ca6ab-872a-49c8-8981-a03db8abe137)

**Comment by queenofcorgis** [comment]
Hi @ozodsilver , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
